src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_x.h

```diff
@@ -417,6 +417,16 @@ namespace sirf {
 			check_data_role(*sptr_ic);
 			sptr_imgs_ = sptr_ic;
 		}
+		// Returns shared pointer to the acquisitions template used. 
```

Comment:
```suggestion
		//! Returns shared pointer to the acquisitions template used. 
```
this way it'll end up in doxygen